### PR TITLE
Update the README of how to benchmark Paddle on Android and add the Chinese version

### LINKED ISSUE #50: Need to add Chinese README of how to benchmark Paddle on Android

### PR DESCRIPTION
**Pull request by Xreki**
Fix #50

### CODE REVIEW

File: benchmark/tool/C/README_cn.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    $ make
+    ```
+
+    执行上述命令执行，会在`build`目录下生成目标可执行文件`inference`。
+
+- **Step 3，准备模型。**
+
+    Android设备上推荐使用`merged model`。以Mobilenet为例，要生成`merged model`，首先你需要准备以下文件：
+    - 模型配置文件[mobilenet.py](https://github.com/PaddlePaddle/Mobile/blob/develop/models/mobilenet.py)，它是使用PaddlePaddle的v2 api编写的`Mobilenet`模型的网络结构。用户可在[models](https://github.com/PaddlePaddle/Mobile/tree/develop/models)获取更多PaddlePaddle常用的网络配置，该repo下面同时提供了使用PaddlePaddle训练模型的方法。
+    - 模型参数文件。使用PaddlePaddle v2 api训练得到的参数将会存储成`.tar.gz`文件。比如，我们提供le一个使用`flower102`数据集训练`Mobilnet`分类模型的参数文件[mobilenet\_flowers102.tar.gz](http://cloud.dlnel.org/filepub/?uuid=4a3fcd7a-719c-479f-96e1-28a4c3f2195e)。你也可以使用以下命令下载该参数文件：
```
Comment (NHZlX) [inline benchmark/tool/C/README_cn.md]: 提供`le` -> 提供了

Comment (Xreki) [reply benchmark/tool/C/README_cn.md]: Done.

File: benchmark/tool/C/README.md [hunk 2]
```diff
@@ -7,45 +7,69 @@ The demo can be run from the command line and can be used to test the inference
...
+    $ adb shell # login Android device
+    odin:/ $ cd /data/local/tmp # switch to the working directory
+    odin:/data/local/tmp $ ls
+    inference  mobilenet_flowers102.paddle
+    odin:/data/local/tmp $ chmod +x inference
+    odin:/data/local/tmp $ ./inference --merged_model ./mobilenet_flowers102.paddle --input_size 150528 # run the executable
+    I1211 17:12:53.334666  4858 Util.cpp:166] commandline:
+    Time of init paddle 3.4388 ms.
+    Time of create from merged model file 141.045 ms.
+    Time of forward time 398.818 ms.
```
Comment (hedaoyuan) [inline benchmark/tool/C/README.md:72]: It is best to indicate this example is running on which phone.

**Review by hedaoyuan — APPROVED** [review]

File: benchmark/tool/C/README.md (hunk 2 above)
Comment (Xreki) [reply benchmark/tool/C/README.md:72]: I just want to show the output of `inference`, the time here is not precise. I will add a comment about this in next PR.

**Pull request merged**